### ISSUE TITLE
QML enum is removed

### ISSUE THREAD

**Issue by onnodb** [jesperhh/qmlfmt#19]
*qmlfmt* is also affected by [QTCREATORBUG-22196](https://bugreports.qt.io/browse/QTCREATORBUG-22196): when trying to save a document such as 
```qml
import QtQuick 2.9

Item {
  enum MyEnum {
    Red,
    Green,
    Blue
  }
}
```
the resulting formatted document has the `enum` removed:
```qml
import QtQuick 2.9

Item {

}
```

**Comment by jesperhh** [comment]
This is to be expected since qmlfmt is using Qt Creator formatting under the hood.

**Comment by onnodb** [comment]
Yes, sorry, I didn't mean to put this issue on your plate. Just thought it'd be useful to put a reference here to the Qt bug report, in case other users of *qmlfmt* encounter this problem.

As a sidenote, as big fans of *clang-format*, we've also been enjoying *qmlfmt* a lot in our project. Thanks for putting this together!

**Comment by jesperhh** [comment]
Great that you like it!

Once a fix is in Qt Creator I will try to get a new version out as soon as possible.

**Comment by jesperhh** [comment]
Looks like this is finally fixed in 4.10, will update the tool once it ships.

**Comment by onnodb** [comment]
Really nice! Thanks!
